### PR TITLE
Modified LogMonitorConfig.json

### PR DESCRIPTION
**Pull request by aneez004**
added a new file location for application logs

### CODE REVIEW

**Comment by profnandaa** [comment]
@aneez004 -- thanks for your contribution. We're onboarding on to the project as maintenance; once we are done with the house-keeping, this will be reviewed promptly.

**Review by profnandaa — APPROVED** [review]
LGTM, thanks for your contribution.

File: LogMonitor/src/LogMonitor/sample-config-files/IIS/LogMonitorConfig.json [hunk 1]
```diff
@@ -22,6 +22,12 @@
         "filter": "*.log",
         "includeSubdirectories": true
       },
+      {
+        "type": "File",
+        "directory": "c:\\wwwroot",
```
Comment (bobsira) [inline LogMonitor/src/LogMonitor/sample-config-files/IIS/LogMonitorConfig.json:27]: The default location for IIS Log files is %SystemDrive%\inetpub\logs\LogFiles. Any reason why you had this as wwwroot?

**Comment by profnandaa** [comment]
Kindly closing this one now that most of the examples are documented in `/docs`.

**Pull request closed without merging**